t/27-consoles-vnc.t

```diff
@@ -243,6 +246,51 @@ subtest 'update framebuffer' => sub {
     $s->set_series(mocked_read => $update_message, $one_rectangle, $ikvm_encoding, $ikvm_specific_data, $actual_image_data);
     $c->update_framebuffer;
     is scalar @printed, 1, 'no further image requested' or diag explain \@printed;
+
+    # Tight encoding, FillCompression
+    $c->_do_endian_conversion($machine_is_big_endian);    # assume server is little-endian
+    $vncinfo = tinycv::new_vncinfo($c->_do_endian_conversion, $c->_true_colour, $c->_bpp / 8, 255, 0, 255, 8, 255, 16);
+    $gray_pixel = pack(CCCC => 31, 37, 41, 0);    # dark prime grey
+    my $of_type_tight_with_coordinates_12_42_4_4 = pack(nnnnNC => 12, 42, 4, 4, 7);
+    my $fill_compression = pack("C", 0x80);
+    $s->set_series(mocked_read => $update_message, $one_rectangle, $of_type_tight_with_coordinates_12_42_4_4, $fill_compression, $gray_pixel);
+    $c->_framebuffer(undef)->width(1024)->height(512)->vncinfo($vncinfo);
+    ok $c->update_framebuffer, 'truthy return value for successful pixel update';
+    ($blue, $green, $red) = $c->_framebuffer->get_pixel(14, 44);
+    is $blue, 41, 'pixel data updated in framebuffer (blue)';
+    is $green, 37, 'pixel data updated in framebuffer (green)';
+    is $red, 31, 'pixel data updated in framebuffer (red)';
```

Comment:
Instead of a comment you can define that a as a title for a nested subtest, e.g.

```suggestion
    subtest 'Tight encoding, FillCompression' => sub {
        $c->_do_endian_conversion($machine_is_big_endian);    # assume server is little-endian
        $vncinfo = tinycv::new_vncinfo($c->_do_endian_conversion, $c->_true_colour, $c->_bpp / 8, 255, 0, 255, 8, 255, 16);
        $gray_pixel = pack(CCCC => 31, 37, 41, 0);    # dark prime grey
        my $of_type_tight_with_coordinates_12_42_4_4 = pack(nnnnNC => 12, 42, 4, 4, 7);
        my $fill_compression = pack("C", 0x80);
        $s->set_series(mocked_read => $update_message, $one_rectangle, $of_type_tight_with_coordinates_12_42_4_4, $fill_compression, $gray_pixel);
        $c->_framebuffer(undef)->width(1024)->height(512)->vncinfo($vncinfo);
        ok $c->update_framebuffer, 'truthy return value for successful pixel update';
        ($blue, $green, $red) = $c->_framebuffer->get_pixel(14, 44);
        is $blue, 41, 'pixel data updated in framebuffer (blue)';
        is $green, 37, 'pixel data updated in framebuffer (green)';
        is $red, 31, 'pixel data updated in framebuffer (red)';
    }
```